people.md

```diff
@@ -0,0 +1,47 @@
+# Steering Council and Institutional Partners
+
+Benevolent Dictator for Life
+============================
+
+Fernando Perez is the Benevolent Dictator for Life (BDFL).
+
+Steering Council
+================
+
+- Brian Granger
+- Damian Avila
+- Fernando Perez
+- Jason Grout
+- Jessica Hamrick
+- Jonathan Frederic
+- Kyle Kelley
+- Matthias Bussionnier
+- Min Ragan-Kelley
+- Thomas Kluyver
+
+NumFOCUS Subcommittee
```

Comment:
I believe the Fiscal oversight team has changed as well, now that most of the group is at Berkeley. From an email with @takluyver, he asked if Jon and I would like to be on the fiscal oversight team, replacing @Carreau and @fperez.